release.sh

```diff
@@ -1,15 +1,13 @@
 # Before you release
-# - install pandoc on your system: apt-get install pandoc
-# - install pypandoc: pip install pypandoc
 # - install twine: pip install twine
-# - bump package version in setup.py
+# - bump package version in setup.cfg
 # - update the package version in the CHANGES file
 # - commit the changes to master and push
 
-PKG_NAME=$(python setup.py --name)
```

Comment:
Is the output of this command just `pycobertura`? If that's the case, we can probably just hardcode it instead of having a code trying to extract it. What do you think?